frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { storagePool } from '../views/multiple-pool';
+
+// Pool var
+const poolName: string = 'example.pool';
+const replicaCount: string = '2';
+
+describe('Test Ceph pool creation', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.clickNavLink(['Storage', 'StorageClasses']);
+    cy.byTestID('item-create').click();
+  });
+
+  after(() => {
+    storagePool.delete(poolName);
```

Comment:
I think we should make this part of the it block as creation is part of it already.

Reply:
ack